openapi.yaml

```diff
@@ -551,15 +551,9 @@ paths:
       - Account
       summary: Credit Card Add/Edit
       description: |
-        Adds/edit credit card information to your Account.
+        **DEPRECATED**. Please use Payment Method Add ([POST /account/payment-methods](/docs/api/account/#payment-method-add)).
 
-        Only one credit card can be associated with your Account, so using this
-        endpoint will overwrite your currently active card information with the
-        new credit card.
-
-        To use this endpoint, you must have a valid `zip` entered for your Account.
-        Use the Account Update ([PUT /account](/docs/api/account/#account-update))
-        endpoint to enter a new zip code.
```

Comment:
This bit is still valid and this endpoint will still report an error if the listed zip is not valid. Should we keep it?

Reply:
I reworded this a bit in the Payment Method Add description to be a bit broader. Agreed that this bit is still valid -- updating that description to be specific about requiring a valid `zip` as well.